teams/teams-ps/teams/Remove-CsTeamsUnassignedNumberTreatment.md

```diff
@@ -0,0 +1,70 @@
+---
+external help file: Microsoft.Open.Teams.CommonLibrary.dll-Help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/powershell/module/teams/remove-csteamsunassignednumbertreatment
+applicable: Microsoft Teams
+author: jenstrier
+ms.author: jenstr
+ms.reviewer: 
+manager:
+schema: 2.0.0
+---
+
+# Remove-CsTeamsUnassignedNumberTreatment
+
+## SYNOPSIS
+Removes a treatment for how calls to an unassigned number range should be routed.
+
+> [!NOTE]
+> **Preview** The use of this cmdlet is in Public Preview.
+  
+## SYNTAX
+
+```powershell
+Remove-CsTeamsUnassignedNumberTreatment -Identity <string>
+```
+
+## DESCRIPTION
+This cmdlet removes a treatment for how calls to an unassigned number range should be routed.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+Remove-CsTeamsUnassignedNumberTreatment -Identity MainAA
+```
+This example removes the treatment MainAA.
+
+
+## PARAMETERS
+
+### -Identity
+The Id of the specific treatment to remove.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+

```

Comment:
```suggestion
### -Force
Specifies whether to suppress warning and confirmation messages. It can be useful in scripting to suppress interactive prompts. If the Force switch isn't provided in the command, you're prompted for administrative input if required.

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

### -WhatIf
Shows what would happen if the cmdlet runs.
The cmdlet is not run.

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases: wi

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -Confirm
Prompts you for confirmation before running the cmdlet.

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases: cf

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### CommonParameters
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```